pulsar-broker/src/test/java/org/apache/pulsar/broker/service/persistent/ShadowReplicatorTest.java

```diff
@@ -131,7 +131,7 @@ public void testShadowReplication() throws Exception {
 
         Message<byte[]> shadowMessage = shadowConsumer.receive(5, TimeUnit.SECONDS);
 
-        Assert.assertEquals(shadowMessage.getData(), sourceMessage.getData());
+        Assert.assertEquals(shadowMessage.getData().length, 0);
```

Comment:
This is a breaking change. The shadow consumer should still receive the message

Reply:
Sorry. I changed it by mistake. The test should fail. However, I found that this test class has never been tested by the CI. Will fix it.